docs/samples/deploying-your-first-bitcoin-dapp.md

```diff
@@ -21,6 +21,10 @@ and [Bitcoin API](https://internetcomputer.org/docs/current/references/ic-interf
         # For rust
         cd examples/rust/basic_bitcoin
 
+   :::note
+   If you choose Rust, you'll need to install [Homebrew](https://brew.sh/) and run `brew install llvm` to be able to compile the example.
```

Comment:
also necessary on linux or is this MacOS-specific?

Reply:
Ah, good point, clarified in the instructions.